src/dataspec/base.py

```diff
@@ -475,8 +542,22 @@ def from_val(
             else:
                 reqkeys[k] = make_spec(v)
 
+        def conform_mapping(d: Mapping) -> Mapping:
```

Comment:
```suggestion
        def conform_mapping(d: Mapping) -> Dict:
```

Reply:
Why do you prefer `Dict` here? `Mapping` is also correct and doesn't indicate mutation is possible.

Side note: If I'm really honest with myself, I think the type annotations in `dataspec` are very lax and unhelpful as it is now. I would love to eventually get to a point where they are comprehensive and meaningful, but that's a larger project.

Reply:
I prefer `Dict` because I thought that, in general, [we want return types to be concrete and argument types to be abstract.](https://docs.python.org/3/library/typing.html#typing.Dict) I don't have a strong opinion and I didn't know you had an annotations overhaul in mind.